x-pack/plugins/endpoint/public/applications/endpoint/view/hosts/index.test.tsx

```diff
@@ -6,40 +6,26 @@
 
 import React from 'react';
 import * as reactTestingLibrary from '@testing-library/react';
-import { Provider } from 'react-redux';
-import { I18nProvider } from '@kbn/i18n/react';
```

Comment:
why remove the i18n bits?  ...cause its a test file and it isn't being tested / translated yet?

Reply:
Hi @EricDavisX . It's removed explicitly from here, but still used via `createAppRootMockRender()`.
In my prior PR I pushed through a custom test renderer that add a `wrapper` around the component being tested that includes all of the context providers that the app uses at runtime. 
See:
- `x-pack/plugins/endpoint/public/applications/endpoint/mocks/app_context_render.tsx:44`, And
- `x-pack/plugins/endpoint/public/applications/endpoint/view/app_root_provider.tsx:23`